recipe/meta.yaml

```
@@ -52,12 +47,13 @@ about:
     of the necessary conversions between Python and C data types, as well as managing the errors occurring in the C
     library.
   license: PostgreSQL
+  license_family: OTHER
   license_file: LICENSE.txt
```

Also, we don't need `recipe/LICENSE.txt` as the source contains `LICENSE` -- and then you should change this line

```suggestion
  license_file: LICENSE
```